src/operator/ndarray_op.cu

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Copyright (c) 2015 by Contributors
+ * \file ndarray_op.cu
+ * \brief
+ * \author Junyuan Xie
+*/
+#include "./ndarray_op-inl.h"
+namespace mxnet {
+namespace op {
+template<>
+Operator* CreateOp<gpu>(NDArrayOpParam param) {
+  return new NDArrayOp<gpu>(param);
+}
+
+template<>
+Context NDArrayOp<gpu>::get_ctx() {
```

Comment:
avoid using cuda file completely, as there is no need to compile GPU code, this also allows touch c++11